spog/ui/src/main.rs

```diff
@@ -7,6 +7,7 @@ mod components;
 mod console;
 mod hooks;
 mod pages;
+mod shared;
```

Comment:
What do you think about instead of creating a new toplevel module here, just putting the simple_pagination inside src/components/pagination.rs ?

Reply:
Yup, good idea I think. Whether you want to do that in this PR or another one I leave up to you.